### ISSUE TITLE
Moonlight freeze on NVIDIA Shield 2019

### ISSUE THREAD

**Issue by nattyduber** [moonlight-stream/moonlight-android#1040]
I'm on latest version of GeForce expérience and on Android TV 11 on Shield 2019
Thèse days i played god of War 2018 and ff7 remake intergrade
Sometime , screen freeze, unplayable. Need to restart Moonlight
When the screen freezing, music is Always ok and i Can listen Kratos speak for example

Why its freezing?
Thanks

**Comment by nattyduber** [comment]
Same problem on Moonlight Windows 11 (surface pro 7) and Xiaomi redmi 9 (miui12.5)

Before, all was ok..

**Comment by nattyduber** [comment]
And 4g or wifi its the same..

**Comment by cgutman** [comment]
On your host PC, disable the hardware-accelerate GPU scheduling option in your Graphics Settings.

**Comment by nattyduber** [comment]
Cgutman : do you know how?
I have a gtx1080 on win11

Thanks

**Comment by nattyduber** [comment]
Oh i think i'have found how disable it.
Thanks i will try this night :)

**Comment by nattyduber** [comment]
Hey , test failed. Same problem with or without hardware-accelerated GPU scheduling option...

**Comment by cgutman** [comment]
Try updating to GeForce Experience 3.25.0.84 and see if that resolves the problem.

**Comment by panic2k** [comment]
I have the same issue with newest Gameready Drivers from 14th and GeForce Experience 3.25.0.84 on Android and Linux Client. I didn't tryed Windows but i expect same behavior.

**Comment by leni8ec** [comment]
Have a same problem on my Shield 2019 TV PRO (Android 11)

**Comment by YABIJIU** [comment]
> On your host PC, disable the hardware-accelerated GPU scheduling option in your Graphics Settings.

it doesn't work.and it influences the Vibration of controler.recover the vibration must reset PC

**Comment by darksidemilk** [comment]
I'm on windows 11, rtx 3080ti. Playing on android over wifi 6 and the host is on a 2.5 gbe lan connection (so over 1 gbps link on client and server). This issue was driving me nuts and kept getting worse. I couldn't even get past the main menu of some games. I gave the gpu scheduling turn off a try. I also disabled the windowed optimizations and auto hdr that were on the same setting screen. Restarted the computer and gave streaming another go. It worked perfectly again. I only played for a few minutes, and it sometimes in the past would freeze after maybe 10-20 minutes or it would freeze every 30 seconds. For sure past the every 30 seconds problem. Thank you for the fix. There's more to test that used to work for me on windows 10 like different monitors and streaming with the monitor off. But very happy that this simple change helped this much, thanks @cgutman!
